clean_all.py

```diff
@@ -67,28 +70,28 @@ def _iterate(
         return instance_list
 
     @staticmethod
+    def _singular_name(name: str) -> str:
+        return name[:-1] if name.endswith("s") else name
```

Comment:
This will not work for all plural forms, but as long as it works for us, then I think it's sufficient.

Reply:
I'm fully aware of that, but it should work in most cases